demo/app/lib/models/connect_issuer_config.dart

```diff
@@ -0,0 +1,36 @@
+import 'dart:ffi';
+
+class ConnectIssuerConfig {
+  final String issuerURI;
+  final List<Object?> scopes;
```

Comment:
Scopes are actually strings, so it should be final List<String> scopes;

Reply:
updated